src/main/java/com/google/cloud/spanner/pgadapter/utils/Metrics.java

```diff
@@ -16,12 +16,11 @@
 import com.google.api.core.InternalApi;
 import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.api.common.Attributes;
-import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.api.metrics.DoubleHistogram;
 import io.opentelemetry.api.metrics.Meter;
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.List;
 
-@InternalApi
```

Comment:
Is it intentional to remove this?

Reply:
I forgot to mention this. This is intended because I need to call this in the tpcc benchmark code.

Reply:
Yeah, but I think that we should still mark it as `@InternalApi`. The annotation is only informational and means that we reserve the right to make (breaking) changes without doing a major version update. So keeping it here won't prevent you from using it in the tpcc benchmark app, but it will tell other potential users that if they do use it, they have to accept that it might change without notice.

Reply:
(I took the liberty of adding the annotation again, we can always reopen the discussion if you feel that we really need to remove this)

Reply:
No problem at all. As long as the tpcc benchmark can use it, I am okay with it.